bm.js

```diff
@@ -10,128 +10,50 @@ const LOCMAP = "locmap";
 const LOCSCORE = "locscore";
 const LOCIMAGE = "locimage";
 
-function locquery(apiKey, location) {
-  console.log(bmUrl(apiKey, LOCQUERY, location))
+function bmQuery(apiKey, arg, queryType) {
```

Comment:
Can we move this method down to the bottom with the other helper methods?

Reply:
yuh